[2023-09-09T02:37:25.688+0000] {processor.py:154} INFO - Started process (PID=45) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:37:25.689+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:37:25.689+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:37:25.689+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:37:25.733+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:37:25.725+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/group_dag_task.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag_task.py", line 3, in <module>
    from groups.group_downloads import download_tasks
  File "/opt/airflow/dags/groups/group_downloads.py", line 3, in <module>
    from airflow.utils.task.group import TaskGroup
ModuleNotFoundError: No module named 'airflow.utils.task'
[2023-09-09T02:37:25.736+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:37:25.753+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.067 seconds
[2023-09-09T02:37:55.885+0000] {processor.py:154} INFO - Started process (PID=120) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:37:55.890+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:37:55.891+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:37:55.891+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:37:55.926+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:37:55.920+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/group_dag_task.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag_task.py", line 3, in <module>
    from groups.group_downloads import download_tasks
  File "/opt/airflow/dags/groups/group_downloads.py", line 3, in <module>
    from airflow.utils.task.group import TaskGroup
ModuleNotFoundError: No module named 'airflow.utils.task'
[2023-09-09T02:37:55.928+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:37:55.946+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.064 seconds
[2023-09-09T02:38:26.075+0000] {processor.py:154} INFO - Started process (PID=195) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:38:26.076+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:38:26.076+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:38:26.076+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:38:26.108+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:38:26.103+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/group_dag_task.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag_task.py", line 3, in <module>
    from groups.group_downloads import download_tasks
  File "/opt/airflow/dags/groups/group_downloads.py", line 3, in <module>
    from airflow.utils.task.group import TaskGroup
ModuleNotFoundError: No module named 'airflow.utils.task'
[2023-09-09T02:38:26.111+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:38:26.130+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.058 seconds
[2023-09-09T02:38:56.537+0000] {processor.py:154} INFO - Started process (PID=260) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:38:56.538+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:38:56.539+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:38:56.539+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:38:56.566+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:38:56.560+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/group_dag_task.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag_task.py", line 3, in <module>
    from groups.group_downloads import download_tasks
  File "/opt/airflow/dags/groups/group_downloads.py", line 3, in <module>
    from airflow.utils.task.group import TaskGroup
ModuleNotFoundError: No module named 'airflow.utils.task'
[2023-09-09T02:38:56.568+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:38:56.585+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.051 seconds
[2023-09-09T02:39:27.360+0000] {processor.py:154} INFO - Started process (PID=334) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:39:27.361+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:39:27.361+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:39:27.361+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:39:27.391+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:39:27.385+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/group_dag_task.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag_task.py", line 3, in <module>
    from groups.group_downloads import download_tasks
  File "/opt/airflow/dags/groups/group_downloads.py", line 3, in <module>
    from airflow.utils.task.group import TaskGroup
ModuleNotFoundError: No module named 'airflow.utils.task'
[2023-09-09T02:39:27.393+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:39:27.410+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.053 seconds
[2023-09-09T02:39:57.698+0000] {processor.py:154} INFO - Started process (PID=408) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:39:57.698+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:39:57.699+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:39:57.699+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:39:57.726+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:39:57.720+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/group_dag_task.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag_task.py", line 3, in <module>
    from groups.group_downloads import download_tasks
  File "/opt/airflow/dags/groups/group_downloads.py", line 3, in <module>
    from airflow.utils.task.group import TaskGroup
ModuleNotFoundError: No module named 'airflow.utils.task'
[2023-09-09T02:39:57.728+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:39:57.744+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.049 seconds
[2023-09-09T02:40:27.843+0000] {processor.py:154} INFO - Started process (PID=482) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:40:27.844+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:40:27.845+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:40:27.844+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:40:27.874+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:40:27.867+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/group_dag_task.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag_task.py", line 3, in <module>
    from groups.group_downloads import download_tasks
  File "/opt/airflow/dags/groups/group_downloads.py", line 3, in <module>
    from airflow.utils.task.group import TaskGroup
ModuleNotFoundError: No module named 'airflow.utils.task'
[2023-09-09T02:40:27.875+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:40:27.893+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.053 seconds
[2023-09-09T02:40:58.131+0000] {processor.py:154} INFO - Started process (PID=547) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:40:58.132+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:40:58.133+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:40:58.133+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:40:58.164+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:40:58.159+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/group_dag_task.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag_task.py", line 3, in <module>
    from groups.group_downloads import download_tasks
  File "/opt/airflow/dags/groups/group_downloads.py", line 3, in <module>
    from airflow.utils.task.group import TaskGroup
ModuleNotFoundError: No module named 'airflow.utils.task'
[2023-09-09T02:40:58.166+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:40:58.185+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.056 seconds
[2023-09-09T02:41:29.061+0000] {processor.py:154} INFO - Started process (PID=631) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:41:29.062+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:41:29.063+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:41:29.063+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:41:29.102+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:41:29.096+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/group_dag_task.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/group_dag_task.py", line 4, in <module>
    from groups.group_transforms import transform_tasks
  File "/opt/airflow/dags/groups/group_transforms.py", line 3, in <module>
    from airflow.utils.task.task_group import TaskGroup
ModuleNotFoundError: No module named 'airflow.utils.task'
[2023-09-09T02:41:29.104+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:41:29.128+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.069 seconds
[2023-09-09T02:41:59.637+0000] {processor.py:154} INFO - Started process (PID=706) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:41:59.638+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:41:59.638+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:41:59.638+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:41:59.679+0000] {processor.py:766} INFO - DAG(s) dict_keys(['group_dag_task']) retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:41:59.784+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:41:59.784+0000] {manager.py:507} INFO - Created Permission View: can read on DAG:group_dag_task
[2023-09-09T02:41:59.795+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:41:59.795+0000] {manager.py:507} INFO - Created Permission View: can delete on DAG:group_dag_task
[2023-09-09T02:41:59.804+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:41:59.804+0000] {manager.py:507} INFO - Created Permission View: can edit on DAG:group_dag_task
[2023-09-09T02:41:59.805+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:41:59.804+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-09-09T02:41:59.815+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:41:59.815+0000] {dag.py:2606} INFO - Creating ORM DAG for group_dag_task
[2023-09-09T02:41:59.824+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:41:59.824+0000] {dag.py:3340} INFO - Setting next_dagrun for group_dag_task to 2023-09-08T00:00:00+00:00, run_after=2023-09-09T00:00:00+00:00
[2023-09-09T02:41:59.846+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.212 seconds
[2023-09-09T02:42:30.236+0000] {processor.py:154} INFO - Started process (PID=784) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:42:30.237+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:42:30.238+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:42:30.238+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:42:30.277+0000] {processor.py:766} INFO - DAG(s) dict_keys(['group_dag_task']) retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:42:30.308+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:42:30.308+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-09-09T02:42:30.328+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:42:30.328+0000] {dag.py:3340} INFO - Setting next_dagrun for group_dag_task to 2023-09-08T00:00:00+00:00, run_after=2023-09-09T00:00:00+00:00
[2023-09-09T02:42:30.347+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.113 seconds
[2023-09-09T02:43:00.562+0000] {processor.py:154} INFO - Started process (PID=853) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:43:00.563+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:43:00.564+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:43:00.564+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:43:00.603+0000] {processor.py:766} INFO - DAG(s) dict_keys(['group_dag_task']) retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:43:00.634+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:43:00.634+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-09-09T02:43:00.655+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:43:00.655+0000] {dag.py:3340} INFO - Setting next_dagrun for group_dag_task to 2023-09-08T00:00:00+00:00, run_after=2023-09-09T00:00:00+00:00
[2023-09-09T02:43:00.673+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.113 seconds
[2023-09-09T02:43:31.125+0000] {processor.py:154} INFO - Started process (PID=927) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:43:31.127+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:43:31.128+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:43:31.127+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:43:31.183+0000] {processor.py:766} INFO - DAG(s) dict_keys(['group_dag_task']) retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:43:31.217+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:43:31.217+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-09-09T02:43:31.243+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:43:31.243+0000] {dag.py:3340} INFO - Setting next_dagrun for group_dag_task to 2023-09-08T00:00:00+00:00, run_after=2023-09-09T00:00:00+00:00
[2023-09-09T02:43:31.265+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.143 seconds
[2023-09-09T02:44:01.307+0000] {processor.py:154} INFO - Started process (PID=1004) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:44:01.308+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:44:01.309+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:44:01.309+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:44:01.342+0000] {processor.py:766} INFO - DAG(s) dict_keys(['group_dag_task']) retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:44:01.374+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:44:01.373+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-09-09T02:44:01.393+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:44:01.393+0000] {dag.py:3340} INFO - Setting next_dagrun for group_dag_task to 2023-09-08T00:00:00+00:00, run_after=2023-09-09T00:00:00+00:00
[2023-09-09T02:44:01.411+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.107 seconds
[2023-09-09T02:44:31.471+0000] {processor.py:154} INFO - Started process (PID=1079) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:44:31.472+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:44:31.473+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:44:31.473+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:44:31.514+0000] {processor.py:766} INFO - DAG(s) dict_keys(['group_dag_task']) retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:44:31.544+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:44:31.544+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-09-09T02:44:31.570+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:44:31.570+0000] {dag.py:3340} INFO - Setting next_dagrun for group_dag_task to 2023-09-08T00:00:00+00:00, run_after=2023-09-09T00:00:00+00:00
[2023-09-09T02:44:31.591+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.123 seconds
[2023-09-09T02:45:01.674+0000] {processor.py:154} INFO - Started process (PID=1141) to work on /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:45:01.675+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/group_dag_task.py for tasks to queue
[2023-09-09T02:45:01.675+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:45:01.675+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:45:01.714+0000] {processor.py:766} INFO - DAG(s) dict_keys(['group_dag_task']) retrieved from /opt/airflow/dags/group_dag_task.py
[2023-09-09T02:45:01.742+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:45:01.742+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-09-09T02:45:01.762+0000] {logging_mixin.py:120} INFO - [2023-09-09T02:45:01.762+0000] {dag.py:3340} INFO - Setting next_dagrun for group_dag_task to 2023-09-08T00:00:00+00:00, run_after=2023-09-09T00:00:00+00:00
[2023-09-09T02:45:01.780+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/group_dag_task.py took 0.109 seconds
